### ISSUE TITLE
[Bug Report] recent table change throws errors on empty cells

### ISSUE THREAD

**Issue by vordimous** [ElemeFE/element#11240]
### Element UI version
2.3.8

### OS/Browsers version
macos chrome

### Vue version
2.5.16

### Reproduction Link
[https://codepen.io/vordimous/pen/VxVmWe](https://codepen.io/vordimous/pen/VxVmWe)

### Steps to reproduce
mouse over the empty cell.

### What is Expected?
not 
```
Uncaught DOMException: Failed to execute 'setEnd' on 'Range': There is no child at offset 1.
    at VueComponent.handleCellMouseEnter (https://unpkg.com/element-ui@2.3.8/lib/index.js:1:225076)
    at mouseenter (https://unpkg.com/element-ui@2.3.8/lib/index.js:1:221134)
    at invoker (https://unpkg.com/vue@2.5.16/dist/vue.js:2029:18)
    at HTMLTableCellElement.fn._withTask.fn._withTask (https://unpkg.com/vue@2.5.16/dist/vue.js:1828:18)
```

### What is actually happening?
```
Uncaught DOMException: Failed to execute 'setEnd' on 'Range': There is no child at offset 1.
    at VueComponent.handleCellMouseEnter (https://unpkg.com/element-ui@2.3.8/lib/index.js:1:225076)
    at mouseenter (https://unpkg.com/element-ui@2.3.8/lib/index.js:1:221134)
    at invoker (https://unpkg.com/vue@2.5.16/dist/vue.js:2029:18)
    at HTMLTableCellElement.fn._withTask.fn._withTask (https://unpkg.com/vue@2.5.16/dist/vue.js:1828:18)
```
caused by https://github.com/ElemeFE/element/pull/11124#issuecomment-390075580

**Comment by vordimous** [comment]
fixed in 2.3.9

**Comment by jikkai** [comment]
#11219
